templates/results-processing.sh

```diff
@@ -148,58 +117,51 @@ function compareBenchmarkWithPrevious() {
-  PREVIOUS_BENCHMARK_DIR_NAME=$(find "${CURRENT_BENCHMARK_TYPE_DIR}" -mindepth 1 -maxdepth 1 -type d -printf '%T@ %f\n' 2>/dev/null | sort -nr | head -n 2 | tail -n 1 | cut -d' ' -f2)
+  local CURRENT_BENCHMARK_DIR_NAME=$(find "${CURRENT_BENCHMARK_TYPE_DIR}" -mindepth 1 -maxdepth 1 -type d -printf '%T@ %f\n' 2>/dev/null | sort -nr | head -n 1 | cut -d' ' -f2)
+  local PREVIOUS_BENCHMARK_DIR_NAME=$(find "${CURRENT_BENCHMARK_TYPE_DIR}" -mindepth 1 -maxdepth 1 -type d -printf '%T@ %f\n' 2>/dev/null | sort -nr | head -n 2 | tail -n 1 | cut -d' ' -f2)
 
-  CURRENT_BENCHMARK_CSV_PATH="${CURRENT_BENCHMARK_TYPE_DIR}/${CURRENT_BENCHMARK_DIR_NAME}/${BENCHMARK_FILE_NAME}"
-  PREVIOUS_BENCHMARK_CSV_PATH="${CURRENT_BENCHMARK_TYPE_DIR}/${PREVIOUS_BENCHMARK_DIR_NAME}/${BENCHMARK_FILE_NAME}"
+  local CURRENT_BENCHMARK_CSV_PATH="${CURRENT_BENCHMARK_TYPE_DIR}/${CURRENT_BENCHMARK_DIR_NAME}/${BENCHMARK_FILE_NAME}"
+  local PREVIOUS_BENCHMARK_CSV_PATH="${CURRENT_BENCHMARK_TYPE_DIR}/${PREVIOUS_BENCHMARK_DIR_NAME}/${BENCHMARK_FILE_NAME}"
 
   if [ ! -d "${VENV_PATH}" ]; then
-    echo "python -m venv ${VENV_PATH}"
     python -m venv "${VENV_PATH}"
   fi
 
   # Activate virtual environment
-  echo "source ${VENV_PATH}/bin/activate"
   source "${VENV_PATH}/bin/activate"
 
-  if [ -f "${NIMBUS_ETH1_GIT_REPO}/scripts/requirements.txt" ]; then
-    echo "pip install -r ${NIMBUS_ETH1_GIT_REPO}/scripts/requirements.txt >/dev/null 2>&1"
-    pip install -r "${NIMBUS_ETH1_GIT_REPO}/scripts/requirements.txt" >/dev/null 2>&1
+  if [ -f "${NIMBUS_ETH1_REPO}/scripts/requirements.txt" ]; then
+    pip install -r "${NIMBUS_ETH1_REPO}/scripts/requirements.txt" >/dev/null 2>&1
   fi
 
-  echo "python ${NIMBUS_ETH1_BLOCKS_IMPORT_SCRIPT_PATH} ${PREVIOUS_BENCHMARK_CSV_PATH} ${CURRENT_BENCHMARK_CSV_PATH} 2>&1 || true"
-
   python "${NIMBUS_ETH1_BLOCKS_IMPORT_SCRIPT_PATH}" "${PREVIOUS_BENCHMARK_CSV_PATH}" "${CURRENT_BENCHMARK_CSV_PATH}" 2>&1 || true
 
   # Deactivate virtual environment
-  echo "deactivate"
   deactivate
 
   echo "=========================="
 }
 
-function moveServiceLogToRepo() {
+function generateSystemdServiceLogs() {
 
-  SYSTEMD_INVOCATION_ID=$(systemctl show --value -p InvocationID nimbus-eth1-mainnet-master-"${BENCHMARKING_TYPE}"-benchmark.service)
-
-  echo "SYSTEMD_INVOCATION_ID for this service run is: ${SYSTEMD_INVOCATION_ID}"
+  local BENCHMARKING_SERVICE_LOG_FILE_NAME="nimbus-eth1-mainnet-master-${BENCHMARKING_TYPE}-benchmark-output.log"
 
   # generate log file for previous run
-  echo "journalctl _SYSTEMD_INVOCATION_ID=${SYSTEMD_INVOCATION_ID} -q --no-hostname >>${BENCHMARK_DESTINATION}/${BENCHMARKING_SERVICE_LOG_FILE_NAME}"
-  journalctl _SYSTEMD_INVOCATION_ID="${SYSTEMD_INVOCATION_ID}" -q --no-hostname >>"${BENCHMARK_DESTINATION}/${BENCHMARKING_SERVICE_LOG_FILE_NAME}"
+  journalctl _SYSTEMD_INVOCATION_ID="${SYSTEMD_INVOCATION_ID}" -q --no-hostname >>"${CURRENT_BENCHMARK_RUN_DIR}/${BENCHMARKING_SERVICE_LOG_FILE_NAME}"
```

Comment:
Unnecessary variable used only once:
```suggestion
  journalctl _SYSTEMD_INVOCATION_ID="${SYSTEMD_INVOCATION_ID}" -q --no-hostname \
    >> "${CURRENT_BENCHMARK_RUN_DIR}/nimbus-eth1-mainnet-master-${BENCHMARKING_TYPE}-benchmark-output.log"
```